data/json/items/ranged/archery.json

```diff
@@ -120,21 +120,6 @@
     "show_stats": true,
     "effects": [ "RECOVER_8" ]
   },
-  {
-    "type": "AMMO",
-    "id": "arrow_metal_sharpened_fletched",
-    "name": { "str": "makeshift metal arrow" },
-    "symbol": "=",
-    "color": "green",
-    "description": "A simple fletched wooden arrow shaft with a sharpened makeshift tip.",
-    "copy-from": "arrow_field_point_fletched",
-    "price": 0,
-    "price_postapoc": 20,
-    "damage": { "damage_type": "stab", "armor_penetration": 1 },
-    "show_stats": true,
-    "effects": [ "RECOVER_20" ],
-    "melee_damage": { "bash": 2 }
-  },
```

Comment:
You have not handled any obsoletion or migration for this.

Reply:
Sorry, I'm new to this. Is there a guide for that? What do I need to do for that?

Reply:
1.
    A. Shove the item's definition(what this comment is under) into `json/obsoletion/obsolete_items.json`
or
    B. Migrate them to makeshift wooden arrows by adding this to `obsoletion_and_migration_0.I/migration_items.json`
```
{
    "id": "arrow_metal_sharpened_fletched",
    "type": "MIGRATION",
    "replace": "arrow_wooden_heavy"
  },
```
2.  Add this to `json/obsoleteion/obsoletion_and_migration_0.I/obsolete-0.I-recipes.json` to get rid of the recipe

```
  {
    "result": "arrow_metal_sharpened_fletched",
    "type": "recipe",
    "obsolete": true
  },
```

Reply:
got it, thank you so much!